ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragmentcontainer/v1/fragmentcontainer/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't this exactly same as adaptive form container, if you add a super resource type and there are no dialog changes, you don't need all of this, it should auto pick from the super resource type. Please remove relevant XML files from dialog, design dialog etc

Reply:
it is different as it doesn't have submit action.

Reply:
In that case please refer it directly from parent resource type, just hide/remove relevant section, right now it is copy pasted.